### PR TITLE
Translate ProxySQL Integration Guide to Chinese

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/11461
- Other reference link(s): https://docs.pingcap.com/tidb/dev/dev-guide-proxysql-integration

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.4 label because the v6.4 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.4 and will no longer receive new updates.

**Comment by Oreoxmt** [comment]
@Icemap https://github.com/Icemap/docs-cn/pull/6 PTAL

File: develop/dev-guide-proxysql-integration.md
Comment (Oreoxmt) [inline develop/dev-guide-proxysql-integration.md:1]: Preview: https://pingcap-docs-website-scaffold-git-preview-pingca-f62548-oreoxmt.vercel.app/zh/tidb/dev/dev-guide-proxysql-integration

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-7.3 label because the v7.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.3 and will no longer receive new updates

File: develop/dev-guide-proxysql-integration.md [hunk 1]
```diff
@@ -1,659 +1,1128 @@
...
 
-    ```shell
-    systemctl start proxysql
+    </SimpleTab>
+
+### 选项 2: 集成本地部署的 TiDB 与 ProxySQL
+
+在这个集成中，你将使用 [TiDB](https://hub.docker.com/r/pingcap/tidb) 和 [ProxySQL](https://hub.docker.com/r/proxysql/proxysql) 的 Docker 镜像设置环境。你也可以尝试[其他方式安装 TiDB](/quick-start-with-tidb.md)。
+
+下面的步骤将在端口 `6033` 和 `4000` 上分别设置 ProxySQL 和 TiDB，请确保这些端口是可用的。
```
Comment (lilin90) [inline develop/dev-guide-proxysql-integration.md:434]:
```suggestion
下面的步骤将在端口 `6033` 和 `4000` 上分别设置 ProxySQL 和 TiDB，请确保这些端口可用。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by lilin90 — COMMENTED** [review]
@Icemap Please resolve the confilct.

**Review by Oreoxmt — APPROVED** [review]

**Comment by Icemap** [comment]
Solved. PTAL.

File: develop/dev-guide-proxysql-integration.md [hunk 2]
```diff
@@ -1,659 +1,1129 @@
...
-运行结果：
+    </div>
 
-```sql
-+--------------------+
-| VERSION()          |
-+--------------------+
-| 8.0.11-TiDB-v7.6.0 |
-+--------------------+
-```
```
Comment (Oreoxmt) [inline develop/dev-guide-proxysql-integration.md:542]:
```suggestion
```

**Pull request merged**